ExamPortal/Models/examDetail.js

```diff
@@ -0,0 +1,19 @@
+const mongoose  = require('mongoose')
+
+const Schema = mongoose.Schema
+
+const examSchema = new Schema({
+	examName: String,
+	examCode: String,
+	facultyId: String,
```

Comment:
What does facultyId mean here? If it's not the correct key name, please change it. It's very misleading.